static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -71,6 +71,10 @@ export type RenderFunctionBaggage = {
   unit?: string;
 };
 
+type RenderFunctionOptions = {
+  disableOnClick?: boolean;
```

Comment:
can we name this in the positive? instead of disable, it's an `enable`

if the value is `undefined` then it'll default to `true`.